src/main/java/seedu/duke/Account.java

```diff
@@ -11,8 +11,8 @@ public class Account {
         balance = initialBalance;
     }
 
-    public float getBalance(){
-        return balance / 100.0f;
+    public int getBalance(){
+        return balance;
```

Comment:
changed to int , cause numbers should stored as int. Only change it to float when printing in UI

Reply:
I think it is ok to keep this, because we still need to divide by 100 eventually. The fix should be in the constructor where we multiply by 100 instead as we intend to multiply by 100 to store in integer and account for the cents. I have did this in my branch

Reply:
I actually edited ur addCommand and balaceCommand to implement this "numbers from input x100, numbers to output /100" thing, so everything should works now. My rationale is , you only /100 when you need to print the balance to UI, other than that , all operation in app should in the form of x100

Reply:
![image](https://user-images.githubusercontent.com/56477330/224881370-43c9d399-0b7a-4e14-bfe0-4448c9c526de.png)

like this part, after your call `getBalance` , you actually need to do some operation which i think should be in `int` better, isnt it?

Reply:
Hmm the purpose of multiplying it in the constructor is for abstraction. As the users, we just need to know that we pass in say SGD 100. Behind the scenes, it will be stored as `balance = 10000` and `currency = SGD`. When we perform a `getBalance`, it will divide the balance by 100 behind the scenes again and get back the SGD 100 which will also be the expected input and output.